contents/handbook/people/team-structure/app-small-teams.md

```diff
@@ -0,0 +1,41 @@
+---
+title: App Small Teams
+sidebar: Handbook
+showTitle: true
+hideAnchor: false
+---
+
+App Small Teams are particular types of Small Teams that own an area of the product.
+
+## What does owning an area of the product mean?
+
+The app small team is responsible for everything related to that area of the product including:
+
+1. Usage
+2. Quality
+3. Revenue
+
+## What actions should the app small teams be doing for their area?
+
+- Each quarter:
+  1. Create good quarterly OKRs
+  2. Create an ambitious vision for their area
+- During the quarter:
+  1. Maintain a prioritized roadmap to help them achieve their objectives
+  2. Speak to customers
+  3. Monitor relevant metrics including those covering Usage, Quality and Revenue
+  4. Triage and fix related bugs
+  5. Assist the support hero in answering related questions
+  6. Collaborate with marketing
+
+## What is the role of the team lead?
+
+The team lead should led the team to make sure the above happens. They should get the team fully involved rather that doing everything themselves.
```

Comment:
🤔 "team lead should lead the team" ?

```suggestion
The team lead is overall responsible for ensuring the above happens. They should focus on enabling the team to solve these tasks together rather than trying to do it all themself.
```